### ISSUE TITLE
[BUG] [Optifine 1.16 pre2] Emissive transparency / brightness doesn't work in 1.15/1.16

### ISSUE THREAD

**Issue by SkaliberWasTaken** [sp614x/optifine#4306]
In previous versions of Optifine you could change the translucency of emissive textures to directly affect the brightness of the emissive. In 1.15 transparency is ignored, and emissive textures can only be at full brightness. Transparency within emissive has also never worked for animations, so the transition between different levels of transparency is sharp.

The video demonstration was too large to upload here, so this is the emissive not working in 1.16 pre2:
https://imgur.com/Pv1gsYH

This video was recorded in 1.14.4, and you can see that the translucency does not change smoothly with the animation, as it should.
https://imgur.com/2ja6j7z

Translucency should change based on opacity within the texture, as can be seen by the varied emissivity on this redstone wire:
![image](https://user-images.githubusercontent.com/50083260/87064499-baa29c00-c207-11ea-8398-bd6dac7419cd.png)

In conclusion the 2 features being requested are as follows:
Texture opacity has a direct effect on emissive brightness levels
Varying levels of emissive translucency should transition smoothly when used in animations

**Comment by betaklerik** [comment]
I'm having a similar issue right now but with armor models and emissive textures, the opacity of the emissive texture is being ignored

**Comment by Awfullyfishy** [comment]
This would really be a good feature and I'm really surprised we're not able to control the brightness of emissives.

Either let the transparency of the emissive overlay control how bright it is or add the feature via emissive.properties so you can for example do:
`suffix.emissive=_fullglow=100`
`suffix.emissive=_faintglow=10`
And this would then let you use two suffixes, one with 100% brightness and one with 10% brightness.

**Comment by SkaliberWasTaken** [comment]
This issue is a direct result of OptiFine moving translucency for most blocks, and can be semi-fixed already by using block.properties.
